packages/paste-website/src/components/shortcodes/ChangelogRevealer.tsx

```diff
@@ -1,17 +1,18 @@
 import * as React from 'react';
+import {Box} from '@twilio-paste/box';
 import {Separator} from '@twilio-paste/separator';
 import {Disclosure, DisclosureContent, DisclosureHeading} from '@twilio-paste/disclosure';
 
 export const ChangelogRevealer: React.FC<{children: NonNullable<React.ReactNode>}> = ({children}) => {
   return (
-    <>
+    <Box id="changelog">
```

Comment:
I might be tempted to make this a little more specific, but otherwise, fine.

Reply:
Thats fair. I'll change to `component-changelog`.